### DISCUSSION TITLE
Could I make an introduction to this project?

### DISCUSSION THREAD

**Discussion by Wudji** [Sinytra/ForgifiedFabricAPI#14] [Q&A] [answered]
Hello! Could I make an introduction to this project (including ForgifiedFabricAPI, Sinytra Connector......) on https://www.mcbbs.net/? I hope it is used by more Chinese players! In the introduction, I'll include your username and the license of the mods. I'll link to Modrinth / CurseForge for the download.

**Comment by Su5eD** [comment, marked as answer]
Yes, absolutely. Good to see the community grow :) If you have any questions about the project feel free to ask them here.

**Reply by Wudji** [reply]
Introduction in Simplified Chinese : https://www.mcbbs.net/thread-1462812-1-1.html
and a video on bilibili.com: https://www.bilibili.com/video/BV15N411876Y
